src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

```diff
@@ -53,6 +53,7 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
   const [newDetail, setNewDetail] = useState('');
   const [show, setShow] = useState(false);
   const risksQuery = useGetRisksForProject(projectId);
+  const { isLoading } = useEditSingleRisk();
```

Comment:
Instead of using useEditSingleRisk() again, you should put the isLoading before the mutate async in line 51 where you can see we are already calling useEditSingleRisk().